lib/ReactViews/Map/Panels/InnerPanel.jsx

```diff
@@ -151,7 +159,7 @@ const InnerPanel = createReactClass({
               }
             }
             ${(p) =>
-              p.showDropdownAsModal &&
+              showDropdownAsModal &&
```

Comment:
This change is causing a white screen when opening map settings and other dropdowns.
<img width="572" alt="Screenshot 2023-10-04 at 13 35 40" src="https://github.com/TerriaJS/terriajs/assets/19208948/6d7dc277-ed3c-4254-b62a-b3c570c38e5c">

Reply:
Good catch. I have re-written these as proper styled components with regression tests (simple render specs)